pyshelf/metadata/yaml_codec.py

```diff
@@ -0,0 +1,23 @@
+import yaml
+
+
+class YamlCodec(object):
+    def serialize(self, metadata):
+        # safe_dump so it doesn't try to represent a python
+        # object in yaml and only serializes native yaml
+        # types.
+        #
+        # default_flow_style so that it doesn't try to stick
+        # inline json for smaller objects.
+        contents = yaml.safe_dump(
+            metadata,
+            encoding="utf-8",
+            indent=4,
+            default_flow_style=False
+        )
+
+        return contents
+
+    def deserialize(self, metadata):
+        metadata = yaml.load(metadata)
+        return metadata
```

Comment:
I typically make separate lines if I am not being lazy but I am not terribly bothered either way.  Its because of debugging.  Here are the two cases

```
return yaml.load(metadata)
# turns into
metadata = yaml.load(metadata)
import pprint; pprint.pprint(metadata)
return metadata
```
Instead of 

```
metadata = yaml.load(metadata)
return metadata
# turns into
metadata = yaml.load(metadata)
import pprint; pprint.pprint(metadata)
return metadata
```

Easier